Vizzuality/marxan-cloud
# Notifications

## Notification system

### Overview

This PR adds:
- Toast Provider
- Toast Container
- Toast Component

You can always get the functions from the toasts by using the `useToasts` hook. 

### Testing instructions
- Try to signin wrong to see an error notification
- Try to edit a project name or scenario name
